doc/source/index.rst

```diff
@@ -1,51 +1,47 @@
-Welcome to PDFmorph's Documentation!
+Welcome to SNMF's Documentation!
 ====================================
 
-``PDFmorph``: An easy way to compare PDFs.
+``SNMF``: Determine structural phase components, their ratios, and how they change with time from
```

Comment:
"This library implements the stretched non negative matrix factorization (sNMF) and sparse stretched NMF (ssNMF) algorithms described in <link to paper and reference to be put here>.  

This algorithm is designed to do an NMF factorization on a set of signals ignoring any uniform stretching of the signal on the independent variable axis.  For example, for powder diffraction data taken from samples containing multiple chemical phases where the measurements were done at different temperatures and the materials were undergoing thermal expansion."